app/src/main/java/com/example/admin/personallibrarycatalogue/data/Book.java

```diff
@@ -0,0 +1,78 @@
+package com.example.admin.personallibrarycatalogue.data;
+
+import android.graphics.drawable.Drawable;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+
+/**
+ * Created by Mikhail Valuyskiy on 25.05.2015.
+ */
+public class Book {
+
+    //region Private fields
+    @Nullable private Integer id_ = null;
+    private String title_;
+    private String author_;
+    private String description_;
+    private byte[] cover_;
```

Comment:
Description and cover are also ```nullable```. Add this annotation to private fields and getters.